tsconfig.json

```diff
@@ -58,5 +58,9 @@
   },
   "include": [
     "./lib/**/*"
+  ],
+  "exclude": [
```

Comment:
I.. think you have to ship `dist`.

This really isn't my area of speciality. Please consult a node expert.